### PR TITLE
docs: add keep-since to TektonConfig docs

### PR DESCRIPTION
**Pull request by avinal**
# Changes

- Adds keep-since to TektonConfig docs
- Type Declaration: https://github.com/tektoncd/operator/blob/main/pkg/apis/operator/v1alpha1/tektonconfig_types.go#L58-L61
- Mutually exclusive keep and keep-since: https://github.com/tektoncd/operator/blob/main/pkg/apis/operator/v1alpha1/tektonconfig_validation.go#L77-L81
- Fixes tektoncd/website#419, RHDEVDOCS-4312
- Blocked by #1025 

Signed-off-by: Avinal Kumar <avinal@redhat.com>

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Run `make test lint` before submitting a PR
- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

# Release Notes

### CODE REVIEW

File: docs/TektonConfig.md [hunk 1]
```diff
@@ -151,8 +151,11 @@ pruner:
 
 - `resources`: supported resources for auto prune are `taskrun` and `pipelinerun`
 - `keep`: maximum number of resources to keep while deleting removing
+- `keep-since`: retain the resources younger than the specified value in minutes
 - `schedule`: how often to clean up resources. User can understand the schedule syntax [here][schedule].
 
+**NOTE**: `keep` and `keep-since` are mutually exclusive. Specify only one of them.
```
Comment (piyush-garg) [inline docs/TektonConfig.md:157]: this is not the case anymore with latest releases

Comment (avinal) [reply docs/TektonConfig.md:157]: I put it by seeing this piece of code: https://github.com/tektoncd/operator/blob/main/pkg/apis/operator/v1alpha1/tektonconfig_validation.go#L77-L81
and this documentation: https://docs.openshift.com/container-platform/4.10/cicd/pipelines/automatic-pruning-taskrun-pipelinerun.html.

Comment (piyush-garg) [reply docs/TektonConfig.md:157]: May be we need to update this in operator, but this has been done in CLI https://github.com/tektoncd/cli/pull/1533

Comment (avinal) [reply docs/TektonConfig.md:157]: Ok, I will open an issue for this.

Comment (avinal) [reply docs/TektonConfig.md:157]: #974

**Comment by avinal** [comment]
Closing in favor of #1025

**Pull request closed without merging**